README.md

```diff
@@ -0,0 +1,96 @@
+</a>
+
+<a href="https://github.com/anoma/juvix/blob/main/LICENSE">
+<img alt="LICENSE" src="https://img.shields.io/badge/license-GPL--3.0--only-blue.svg" />
+</a>
+
+<a href="https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=455254004">
+<img height="20pt" alt="Open the Juvix Standard Lib in Github Codespace" src="https://github.com/codespaces/badge.svg" />
+</a>
+
+<a href="https://github.com/anoma/juvix">
+<img align="right" width="270" alt="Juvix Mascot" src="assets/images/tara-seating.svg" />
+</a>
+
+Juvix is an open-source, constantly evolving functional programming language
+designed for writing privacy-preserving decentralized applications. Using Juvix, developers can write high-level programs which can be compiled to WASM directly, or through [VampIR](https://github.com/anoma/vamp-ir) to circuits for private execution with [Taiga](https://github.com/anoma/taiga) on Anoma or Ethereum.
+
+## Getting Started
+
+To get started with Juvix, head over to the [documentation website](https://docs.juvix.org) to learn more about the language and its features. You can also find installation instructions and tutorials to help you get started with writing Juvix programs.
+You can download the [latest release](https://github.com/Juvix-lang/juvix/releases) from the Juvix GitHub repository or use the web-based development environment, [Juvix Github
+Codespace](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=102404734&machine=standardLinux32gb&location=WestEurope), which provides a pre-configured workspace ready to use
+with Juvix and the Haskell toolchain installed.
+
+## The Juvix programming language
+
+Juvix provides a high degree of assurance. The Juvix compiler runs
+several static analyses which guarantee the absence of runtime errors.
+Analyses performed include termination, arity, and type checking. As a
+result, functional programs, especially validity predicates, can be
+written with greater confidence in their correctness.
+
+Some language features in Juvix include:
+
+- unicode syntax
+- parametric polymorphism
+- inductive and parametric data types
+- higher-order functions
+- implicit arguments
+- holes in expressions
+- axioms for non-computable terms
+
+The Juvix module system allows developers to break down their programs into smaller, reusable modules that can be compiled separately and combined to create larger programs. These modules can be used to build libraries, which can then be documented using Juvix's built-in documentation generation tool, see for example, [the Juvix standard
+library's website](https://anoma.github.io/juvix-stdlib/). For further
+details, please refer to [the Juvix
+book](https://anoma.github.io/juvix/) which includes our [latest
+updates](https://anoma.github.io/juvix/changelog.html).
+
+## Related projects
+
+If you're interested in Juvix, you may also want to explore the following related projects:
+
+<!-- prettier-ignore -->
+| Project | Description |
```

Comment:
is there a way to format md tables?

Reply:
Yes, I dont' follow. Are you asking because the current format bothers you, or
because of the comment above the table?

Reply:
The rendered table is fine, I am asking if the markdown table can be formatted to look prettier. It's not that important

Reply:
I see. Yes, there's a formatter that add extra "-" to make the table nicer. I'll adress that in another PR, for this and other files.